data/operators/amenity/vending_machine.json

```
@@ -29,6 +29,64 @@
         "operator:wikidata": "Q60790331",
         "vending": "parking_tickets"
       }
+    },
+    {
+      "displayName": "Mera-Serwis SGL (ZTM Warszawa)",
+      "locationSet": {"include": ["pl"]},
+      "tags": {
+        "amenity": "vending_machine",
+        "vending": "public_transport_tickets",
+        "operator": "Mera-Serwis SGL",
+        "network": "network=ZTM Warszawa",
```

This looks like a typo?
```suggestion
        "network": "ZTM Warszawa",
```

---

Yes, thank you!